### ISSUE TITLE
Is it possible to use s5cmd to upload file from url to the s3 bucket?

### ISSUE THREAD

**Issue by tonyman19** [peak/s5cmd#641]

**Comment by ahmethakanbesel** [comment]
Hi,

You can use tools like `curl` or `wget` to download files and pipe them to `s5cmd` using its pipe command. It will be available in version 2.2.0, but you can start using it by compiling from the source code.

```bash
curl https://speed.hetzner.de/1GB.bin | ./s5cmd pipe s3://bucket/1GB.bin
```
or
```bash
wget https://speed.hetzner.de/1GB.bin | ./s5cmd pipe s3://bucket/1GB.bin
```

**Comment by tonyman19** [comment]
Hi Ibrahim,

the problem with piping is that you cannot do the multistream download / upload anymore.
It would be nice when s5cmd would support multistream url downloading.
